notebooks/data_downloader.py

```diff
@@ -0,0 +1,12 @@
+import os
+import git
+from git.repo.base import Repo
+cwd = os.getcwd()
+## we are creating a folder with the name 'data' inside your current working directory. 
+path_to_save = cwd+'/data'
+## Below we are cloning the remote GitHub repo inside the data folder.
+try:
+    Repo.clone_from("https://github.com/SankBad/configuration_datasets.git", path_to_save)
```

Comment:
Do you want to clone that entire repo into this one or do you just want a few of the data sets? You could use `wget.download(https://github.com/tianyin/configuration_datasets/blob/master/configfiles/hadoop+hbase/metadata_q.csv)` to get only the relevant data needed instead of the whole repo.

Reply:
Hi @MichaelClifford , 

I am downloading all the files but not a single CSV file. Hence, wget.download will not work.

Reply:
Do you use all of these files? It a little unclear to me where these all come in to play, is there a notebook or an issue you can link to that describes the need for this entire repo of datasets?

Reply:
Just tag you in a PR. It's still under review. 

Or you can take a look at the data_dir variable of this [notebook](https://github.com/SankBad/configuration-files-analysis/blob/data_type_error_notebook/notebooks/Misconfiguration_detection_framework_for_data_type_errors.ipynb)

Reply:
`https://github.com/SankBad/configuration_datasets/tree/master/configfiles/mysql/v-5.x` so this is only a subset of that repo, correct? In that case why not just grab that one folder instead of the whole repo?

Reply:
Since you have your own fork of this repo, you could also create the folder as a single zip file and add it to your fork to make downloading it simpler.

Reply:
Thanks @MichaelClifford, 
I made the suggested changes.